@navikt/core/react/src/alert/Alert.tsx

```diff
@@ -109,7 +115,12 @@ export const Alert = forwardRef<HTMLDivElement, AlertProps>(
         )}
       >
         <Icon variant={variant} className="navds-alert__icon" />
-        <BodyLong as="div" size={size} className="navds-alert__wrapper">
+        <BodyLong
+          as="div"
+          size={size}
+          className="navds-alert__wrapper"
+          style={contentMaxWidth ? { maxWidth: "43.5rem" } : undefined}
```

Comment:
This change will prob break some applications since inline-styles are the most specific when it comes to CSS-overrides. Maybe something like this could be non-breaking
```
className={cl("navds-alert__wrapper", {"navds-alert__wrapper--no-maxwidth": !contentMaxWidth})}

:where(.navds-alert__wrapper:not(.navds-alert__wrapper--no-maxwidth)) {
  max-width: 43.5rem;
}
```

Since `:where` is used it will have 0 specificity even when using `not` (not tested here so might not be 100% correct)

Reply:
No could probably simplify it 👍  As long as targeting `navds-alert__wrapper` still allows those who have overwritten it now won't notice any changes.